chaosk8s/secret/probes.py

```diff
@@ -0,0 +1,64 @@
+from chaoslib.exceptions import ActivityFailed
+from chaoslib.types import Secrets
+from kubernetes import client
+from logzero import logger
+
+from chaosk8s import create_k8s_api_client
+
+__all__ = ["secret_exists"]
+
+
+def secret_exists(
+    name: str = None,
+    ns: str = "default",
+    label_selector: str = None,
+    raise_if_non_existing: bool = True,
```

Comment:
Since this is a new set of probes, I would like to do without this flag and always return a bool.

Other probes have that flag because we started with raising until someone rightly pointed out thtis created confusion in the journal (it looks like the probe failed when it didn't really).

I think it's better if we simply return a bool for newer probes.

unless you have a need for this probe to raise by default?

Reply:
I blatantly copied the skeleton from `service` and simply left most of it as is. I went ahead and removed the flag.